app/scripts/controllers/transactions/pending-tx-tracker.js

```diff
@@ -211,7 +211,7 @@ export default class PendingTransactionTracker extends EventEmitter {
     const { hash: txHash, txParams: { nonce, from } } = txMeta
     const networkNextNonce = await this.query.getTransactionCount(from)
 
-    if (parseInt(nonce, 16) >= parseInt(networkNextNonce, 16)) {
+    if (parseInt(nonce, 16) >= parseInt(networkNextNonce, 10)) {
```

Comment:
If this is always a BN, we could call `.toNumber()`:

```suggestion
    if (parseInt(nonce, 16) >=networkNextNonce.toNumber()) {
```

This might be clearer (and possibly faster?).

Reply:
I'm fine with either one.